test/frontend_testing/stringInputValidationTest.js

```diff
@@ -14,7 +14,7 @@ function setUp(){
     notResults.insertBefore(mochaDiv, notResults.firstChild);
 }
 
-function stringInputValidationTest() {
+function _rStringInputValidationTest() {
```

Comment:
If we want to establish "_r" as a prefix meaning "this is a front-end testing function," we'd better document that somewhere. Let's add a README file to the test folder, at the very least.

Reply:
yes, I forgot to attach in the comment but I wrote a guideline for frontend testing and it's in the SPOT folder too. I will put the link below.